examples/simple_editor.rs

```diff
@@ -0,0 +1,110 @@
+#![cfg_attr(not(feature = "gtk_3_10"), allow(unused_variables, unused_mut))]
+
+extern crate time;
+extern crate gtk;
+extern crate gdk;
+extern crate cairo;
+extern crate lspace;
+
+use std::string::String;
+use std::rc::Rc;
+
+use gtk::traits::*;
+use gtk::signal::Inhibit;
+
+// for a list of key names see: http://gtk-rs.org/docs/gdk/enums/key/index.html
```

Comment:
Is this comment useful? Surely if someone needs to know this they should look at the `gdk::enums::key` library?